flocker/node/agents/gce.py

```diff
@@ -350,7 +353,7 @@ def _disk_resource_description(self):
         """
         return u"flocker-v1-cluster-id: " + unicode(self._cluster_id)
 
-    def _do_blocking_operation(self, function, **kwargs):
+    def _do_blocking_operation(self, function, max_wait=60, **kwargs):
```

Comment:
:grey_question: Nitpick: timeout is a more typical way of expressing max_wait.

Yes, I had to find _something_ in here :)